### DISCUSSION TITLE
Problem connecting to the freedb server

### DISCUSSION THREAD

**Discussion by Holly-MD** [enzo1982/freac#288] [Help & Support] [answered]
![image](https://user-images.githubusercontent.com/101882406/159025103-53dff24b-fe8a-4201-8a48-df01055c7d51.png)
Anyone else getting an error when querying the database for CD content?

**Comment by djfake** [comment]
I also have this problem this morning.

**Comment by enzo1982** [comment, marked as answer]
It's possible that the GnuDb database was down earlier today. Right now it's working fine for me.

However, the GnuDb project is currently blocking Russian IPs in response to the invasion of Ukraine. So if you happen to be in Russia, that may be the reason you cannot access the database.

In that case, you can try setting the CDDB server to freedb.dbpoweramp.com instead.

**Reply by Holly-MD** [reply]
Thank you!  Their server must have been down as I live in Maryland.  Working fine now.  I've made the change you suggested anyway.
